client/state/google-my-business/reducer.js

```diff
@@ -5,13 +5,15 @@
  */
 import { combineReducers, createReducer, keyedReducer } from 'state/utils';
 import {
+	GOOGLE_MY_BUSINESS_STATS_FAILURE,
 	GOOGLE_MY_BUSINESS_STATS_RECEIVE,
 	GOOGLE_MY_BUSINESS_STATS_REQUEST,
 } from 'state/action-types';
 
 const stats = createReducer( null, {
 	[ GOOGLE_MY_BUSINESS_STATS_RECEIVE ]: ( state, { data } ) => data,
 	[ GOOGLE_MY_BUSINESS_STATS_REQUEST ]: () => null,
+	[ GOOGLE_MY_BUSINESS_STATS_FAILURE ]: () => false,
```

Comment:
I think we usually create an error reducer in those cases.
See https://github.com/Automattic/wp-calypso/blob/master/client/state/login/reducer.js#L108

It's cleaner than setting the data reducer to `false` imo. We can then user a selector to return the error.